go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/dgraph-io/badger v1.6.2
 	github.com/jrick/logrotate v1.0.0
 	github.com/kevinburke/nacl v0.0.0-20190829012316-f3ed23dbd7f8
+	github.com/lib/pq v1.10.4 // indirect
```

Comment:
Instead of an indirect require, use a replace at the end of this file so it can be removed when it's no longer needed. Also makes it easy to add a readable comment explaining why the replace is being done.
```
// Older versions of github.com/lib/pq are required by politeia (v1.9.0)
// and dcrdex (v1.10.3) but only v1.10.4 and above can be compiled for
// the android OS using gomobile. This replace can be removed once any
// of those projects update their github.com/lib/pq dependency.
replace github.com/lib/pq => github.com/lib/pq v1.10.4
```

Remember to run `go mod tidy` after editing the go.mod file.

Also, it'll be find to include a rewording of the comment suggested above in this PR description, to provide more information on why this PR is needed. The current description `This PR resolve the issue can not build lib for android` doesn't say why there's currently a build issue or what is done to fix the issue.